test/test.kitchen.ts

```diff
@@ -20,13 +20,24 @@ import {ncp} from 'ncp';
 import * as pify from 'pify';
 import * as tmp from 'tmp';
 
-const rename = pify(fs.rename);
+const unlink = pify(fs.unlink);
 const ncpp = pify(ncp);
 const keep = !!process.env.GANC_KEEP_TEMPDIRS;
 const stagingDir = tmp.dirSync({keep, unsafeCleanup: true});
 const stagingPath = stagingDir.name;
 const pkg = require('../../package.json');
 
+function copyFile(src, dst) {
```

Comment:
Is there a reason we can't use `fs.copyFile` here?
https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_flags_callback

Either way, we need a comment explaining why we re-invented this method, along with that aforementioned link to the nodejs/node issue.

Reply:
We cannot, fs.copyFile is added in 8.5.0. I actually tried using it but CI quickly explained that I was wrong :)

Reply:
Removed this code - now using `mv` module.